torch_xla/__init__.py

```diff
@@ -63,8 +52,6 @@ def _setup_default_env():
   _set_missing_env('GRPC_VERBOSITY', 'ERROR')
   _set_missing_env('ALLOW_MULTIPLE_LIBTPU_LOAD', '1')
   _set_missing_env('TPU_ML_PLATFORM', 'PyTorch/XLA')
-  if server_is_alive():
```

Comment:
is there a need to keep `_setup_grpc()` around if we are dropping XRT?

cc @will-cromar

Reply:
Thanks, Will has updated in https://github.com/pytorch/xla/pull/5240